x-pack/plugins/infra/public/components/logging/log_entry_flyout/log_entry_flyout.tsx

```diff
@@ -110,19 +137,36 @@ export const LogEntryFlyout = ({
             <EuiTitle size="s">
               <h3 id="flyoutTitle">
                 <FormattedMessage
-                  defaultMessage="Log event document details"
+                  defaultMessage="Details for log entry {logEntryId}"
                   id="xpack.infra.logFlyout.flyoutTitle"
+                  values={{
+                    logEntryId: flyoutItem ? <code>{flyoutItem.id}</code> : '',
```

Comment:
Shall we use `<EuiCode>` here?

Reply:
I considered that, but decided against it, because `<EuiCode>` is a pretty complex component that performs language-specific syntax highlighting on the content. It felt like overkill here, since we're not displaying code but only want to stylize the immutable values.